### PR TITLE
Fix some issues with list fragment

### PR DESCRIPTION
**Pull request by mpourismaiel**
**What this PR does / why we need it**:
- Align date in list fragment to the right of each page

image

- Change size of page title

**Special notes for your reviewer**:

**Release note**:
```release-note
- list: Change alignment of date in list mode
- list: Change page title size based on visibility of summary
```

### CODE REVIEW

**Comment by stp-ip** [comment]
Alignment looks weird: https://deploy-preview-378--syna-demo.netlify.com/dev/list/

Also smaller screens have some isses:
* categories "overlap"
* margin between date, title and categories looks weird

screenshot from 2018-09-23 19-31-12

**Comment by mpourismaiel** [comment]
I keep forgetting about the whitespace issue in production.

**Comment by mpourismaiel** [comment]
The conflict is really hard to fix.

**Comment by mpourismaiel** [comment]
I think the conflict is fixed. We had a lot of changes in the list fragment. Self note to don't change a fragment in two different branches, specially the list fragment and fragments helper.

Please check as many details as possible. I tested it for half an hour but still there might be something that got lost in the conflict.

**Comment by stp-ip** [comment]
Only thing that seems wrong is that display_date is not false by default with tiled, but true. That's in contradiction to our docs and also a change in behaviour.

**Comment by mpourismaiel** [comment]
Fixed. Sorry forgot it's default value.

File: layouts/partials/fragments/list.html [hunk 1]
```diff
@@ -77,16 +77,16 @@ <h5 class="col-12 col-md px-0">
                             </a>
                           </h5>
                         {{- end -}}
-                        {{- if and (not $self.Params.tiled) (ne $self.Params.display_date false) }}
+                        {{- if and (not $self.Params.tiled) (eq $self.Params.display_date false) }}
```
Comment (stp-ip) [inline layouts/partials/fragments/list.html]: Shouldn't that just be `{{- if and (not $self.Params.tiled) $self.Params.display_date }}`?

File: layouts/partials/fragments/list.html [hunk 2]
```diff
@@ -77,16 +77,16 @@ <h5 class="col-12 col-md px-0">
...
                           {{- range $content_page -}}
                             {{- partial "helpers/publish-date.html" (dict "root" . "background" $bg "badge" false) -}}
                           {{- end -}}
                         {{- end }}
                       </div>
                     </div>
-                    {{- if or (and $self.Params.tiled (ne $self.Params.display_date false)) .Params.categories -}}
+                    {{- if or (and $self.Params.tiled (eq $self.Params.display_date false)) .Params.categories -}}
```
Comment (stp-ip) [inline layouts/partials/fragments/list.html]: Shouldn't that just be `{{- if or (and $self.Params.tiled $self.Params.display_date) .Params.categories -}}`

File: layouts/partials/fragments/list.html [hunk 3]
```diff
@@ -77,16 +77,16 @@ <h5 class="col-12 col-md px-0">
...
                       <div class="col-12 pb-1 px-0">
-                        {{- if and $self.Params.tiled (ne $self.Params.display_date false) -}}
+                        {{- if and $self.Params.tiled (eq $self.Params.display_date false) -}}
```
Comment (stp-ip) [inline layouts/partials/fragments/list.html]: Shouldn't that just be `{{- if and $self.Params.tiled $self.Params.display_date -}}`

**Comment by stp-ip** [comment]
Noo found something else being broken :(
https://deploy-preview-378--syna-demo.netlify.com/dev/list/section-1/
https://deploy-preview-378--syna-demo.netlify.com/dev/list/section-2/

**Comment by stp-ip** [comment]
Mhh seems semi intentional. But should it work like that? Aka not having it's own list on these section pages?

**Comment by mpourismaiel** [comment]
That was kind of intentional and it was so from the beginning. The point of it being to have less files for the dev section. I think that's not needed anymore now that we have colors section on the horizon.

**Comment by stp-ip** [comment]
OK let's merge and reiterate.

**Pull request merged**